ReleaseNotes/eng/Altibase_dataCompJ_7_2_Release_Notes.md

```diff
@@ -0,0 +1,105 @@
+Also, it offers an efficient feature for synchronizing two databases in the event of data inconsistency. A proper use of Altibase dataCompJ would be a cornerstone of effective data management with advantageous usability along with its advanced technical performance.  
+
+### 1.1 Hardware Requirements
+
+- Main memory: Minimum 512MB, 4GB or more recommended
+- Disk: More than 500MB of free space for installation
+
+### 1.2 Supported Operating Systems and Platforms
+
+| Mode | JRE                                                         |
+| ---- | ----------------------------------------------------------- |
+| CLI  | Oracle, OpenJDK or IBM Java Runtime Environment 8 or higher |
+
+## 2. Release Information
+
+### 2.1 Version
+
+- 7.2
+
+### 2.2 New Features
+
+- N/A
+
+### 2.3 Bug Fixed
+
+| PK        | SYNOPSIS                                                     |
+| --------- | ------------------------------------------------------------ |
+| BUG-45222 | Requires handling of user-entered object names enclosed in double quotation marks. |
+| BUG-46675 | Error message for the unsupported DB JDBC URL should be provided more clearly. |
+| BUG-46689 | Need to modify the DB type check routine using the JDBC URL string. |
+| BUG-49501 | dataCompJ log4j upgrade to version 2.17.1 due to security issues. |
+
+### 2.4 Database Compatibility
+
+#### Master Database
+
+- Altibase: Altibase 5.3.3 or higher
+
+#### Slave Database
+
+- Oracle: Oracle 9i or higher
+- MariaDB: MariaDB 5.5.x or higher
+
+### 2.5 Properties
+
+### 2.6 Error Messages
+
+## 3. Open Source Libraries Used
+
+dataCompJ is based on the following open-source libraries. The licenses are distributed in a text file format
+along with dataCompJ.
+
+| Library             | Open Source License                                          |
+| ------------------- | ------------------------------------------------------------ |
+| Apache Commons CLI  | http://commons.apache.org/proper/commons-cli/<br/>License: Apache License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt) |
+| Apache Commons Lang | http://commons.apache.org/proper/commons-lang/ <br/>License: Apache License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt) |
+| Apache Commons IO   | http://commons.apache.org/proper/commons-io/ <br>License: Apache License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt) |
+| JDOM                | http://www.jdom.org/ <br/>License: Apache-style Open Source License (http://www.jdom.org/docs/faq.html#a0030) |
+| Log4j               | http://logging.apache.org/index.html <br/>License: Apache License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt) |
+| Oracle JDBC Driver  | http://www.oracle.com <br>License: Oracle Free Use Terms and Conditions (https://www.oracle.com/downloads/licenses/oracle-free-license.html) |
+
+## 4. Packages
+
+| Archive Name                             |
+| ---------------------------------------- |
+| dataCompJ7.2.zip<br/>dataCompJ7.2.tar.gz |
```

Comment:
zip 은 Windows 용이고 tar.gz 은 Linux/Unix 용도인가요? 
맞다면 왼쪽 열 추가해서 구분해주면 좋을 것 같습니다.

Reply:
zip이나 tar는 압축 포멧의 차이이고 OS는 무관합니다.

Reply:
일전에 말씀드린 것처럼 UX팀에서는 릴리즈 노트와 매뉴얼은 모두 영문 기준으로만 제작해서 한글판은 따로 없습니다. 한글판도 필요할까요?

Reply:
> zip이나 tar는 압축 포멧의 차이이고 OS는 무관합니다.

그렇다면, 아래와 같은 안내 설명을 추가해주는 것이 어떨지요? 
 
설치 패키지는 두 가지 압축 형식으로 제공한다. OS에 관계없이 사용할 수 있다.

Reply:
다른 릴리즈 노트 (Migration Center)와 일관성을 위해 아래와 같이 수정하는 걸 어떨가 합니다. 

1.2 Supported Operating Systems and Platforms
...
dataCompJ is a pure Java-based client application relying on the JAVA Runtime Environment (JRE) instead of the client's hardware or an operating system.

Reply:
zip이나 tar에 대해서 굳이 설명을 안 남기려고 하는 이유는, 일반적으로 인지하는 내용에 대해서 설명을 하기 시작하면 내용이 너무 많아질 것 같아서 입니다. 친절한 건 좋지만, 유지보수 비용도 고려해 trade-off를 하는 게 좋은 방법이 아닐까 합니다.

Reply:
> 일전에 말씀드린 것처럼 UX팀에서는 릴리즈 노트와 매뉴얼은 모두 영문 기준으로만 제작해서 한글판은 따로 없습니다. 한글판도 필요할까요?

릴리즈 노트 한글화 관련해서 TASK-7388에서 문제가 제기되고 
[REVIEW-14093 유틸리티 부분 릴리즈 노트 한글화 및 웹 구조 변경 검토 요청](https://altra.altibase.com/altimis-2.0/app_review/review_view.jsp?id=14093) 에서 논의한 적 있습니다. 그 결과로 altiShapeLoader 릴리즈 노트는 한글화 작업을 진행했습니다. 

REVIEW-14093 을 다시 보니 정리가 부족했던 것 같습니다. 구조 변경에만 신경 쓴 듯 요.

Reply:
> zip이나 tar에 대해서 굳이 설명을 안 남기려고 하는 이유는, 일반적으로 인지하는 내용에 대해서 설명을 하기 시작하면 내용이 너무 많아질 것 같아서 입니다. 친절한 건 좋지만, 유지보수 비용도 고려해 trade-off를 하는 게 좋은 방법이 아닐까 합니다.

제가 설치 파일을 압축 파일로 제공하는 경우 Linux/Unix는 tar.gz 형식 Windows 용은 zip 형식으로 제공하는 경우를 종종 봐서 혼동한 듯 합니다. 
@haeinnmin 혜인님 의견이 궁금합니다.